test/chpldoc/compflags/author/withoutAuthor.doc.prediff

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+grep Copyright $2 > $2.prediff.tmp
```

Comment:
Note that because you will have multiple tests in this directory by the time both PRs are merged, and you are using the default docs location, this may pick up the results of other runs if something happens to go wrong with the current test.  I would recommend setting a different documentation output directory and *only* checking that directory for your result.

Otherwise, this change looks right to me!

Reply:
What about putting them in separated subdirs? I think it will be more clear.

Reply:
I don't have any problems with that, but grep should still only search the generated docs directory

Reply:
I just think with catfiles, only the docs/modules/withoutAuthor.doc.html will be passed to grep and it should be fine. Have I missed something?

Reply:
Oh dur, you're right!  I misread the command, sorry about that

Reply:
I guess no changes are needed then, lemme run this through testing :)